toolkit/tools/boilerplate/boilerplate.csv

```diff
@@ -0,0 +1 @@
+boilerplate.csv,execution,"print 'Hello, World!'",12.618944ms,Wed Aug  3 14:52:57 PDT 2022,Wed Aug  3 14:52:57 PDT 2022
```

Comment:
Is this file here by design or just accidentally added?

Reply:
We will remove this csv file. It will be produced if we run boilerplate.go.